server.js

```diff
@@ -61,13 +61,20 @@ function searchBooks(request, response){
       const bookList = superagentResults.body.items.map(book => {
         return new Book(book);
       })
+      console.log(bookList);
       response.send(bookList);
     })
+    // .catch(error => {
+    //   errorHandler(error, request, response);
+    // });
 }
 
 // ========== Book Constructor Object ==========
 function Book(infoFromAPI){
-  this.title = infoFromAPI.volumeInfo.title || 'No title available';
+  this.author = infoFromAPI.volumeInfo.authors || 'no author available';
+  this.title = infoFromAPI.volumeInfo.title || 'no title available';
+  this.description = infoFromAPI.volumeInfo.description || 'no description available';
+  this.imgUrl = infoFromAPI.volumeInfo.imageLinks.thumbnail || 'no image available';

```

Comment:
This property is going to be used for the src property of an image tag. I recommend having a default static image that somehow communicates that an actual image was not provided/available from the search results. Also, you will need to deal with some of the results having 'http' vs 'https' links.